src/app/starredentries/starredentries.component.html

```diff
@@ -81,12 +82,42 @@
           </div>
         </mat-tab>
 
-        <ng-template #workflowSummary let-workflow="workflow" let-isAppTool="isAppTool">
+        <mat-tab>
+          <ng-template mat-tab-label>
+            <div fxLayoutGap="0.5rem">
+              <img class="notebook-icon" src="../../../assets/svg/sub-nav/notebooks.svg" alt="notebook logo" />
+              <span>Notebooks</span>
+              <span class="tab-display" data-cy="starred-notebooks-count">{{ starredNotebooks?.length }}</span>
+            </div>
+          </ng-template>
+          <div>
+            <mat-card class="alert alert-info starred-header" role="alert" *ngIf="starredNotebooks?.length === 0">
+              <mat-icon>warning</mat-icon>
+              You have no starred notebooks.
+            </mat-card>
+            <div fxLayout="column" fxLayoutGap="1rem" class="mb-2 mx-1">
+              <mat-card fxFlex *ngFor="let notebook of starredNotebooks; let last = last" class="m-1 pb-3">
+                <ng-container
+                  *ngTemplateOutlet="workflowSummary; context: { workflow: notebook, isAppTool: false, isNotebook: true }"
+                ></ng-container>
+              </mat-card>
+            </div>
+          </div>
+        </mat-tab>
+
+        <ng-template #workflowSummary let-workflow="workflow" let-isAppTool="isAppTool" let-isNotebook="isNotebook">
           <div fxLayout="row" fxLayoutGap="10px" fxLayoutAlign="space-between">
             <a [routerLink]="(isAppTool ? entryType.AppTool : entryType.BioWorkflow) | routerLink: workflow" class="no-underline">
               <mat-card-header>
                 <mat-card-title fxLayoutAlign="space-between" fxLayoutGap="1rem">
                   <img
+                    *ngIf="isNotebook"
+                    src="../../../assets/svg/sub-nav/notebooks.svg"
```

Comment:
If you changed the name of this `notebooks.svg` to singular `notebook.svg` and you followed Steve's suggestion of using `entryType` and getting rid of `isAppTool` and `isNotebook` in this [comment](https://github.com/dockstore/dockstore-ui2/pull/1726/files#r1138091424), I think that you can get rid of the new `img` block and modify the existing one to get the image `src` and `alt` from `entryType` because [EntryType](https://github.com/dockstore/dockstore-ui2/blob/fa45f93c742f44335c10c43542e713b495d897bc/src/app/shared/enum/entry-type.ts#L11) is already the singular word.

Something like:
```
<img
  src="../../../assets/svg/sub-nav/{{ entryType }}.svg"
  alt="{{ entryType }} logo"
/>